native/cocos/renderer/pipeline/reflection-probe/ReflectionProbeFlow.cpp

```diff
@@ -64,19 +64,33 @@ void ReflectionProbeFlow::render(scene::Camera *camera) {
     const auto *sceneData = _pipeline->getPipelineSceneData();
     const auto probes = scene::ReflectionProbeManager::getInstance()->getAllProbes();
     for (auto * probe : probes) {
-        if (probe->needRender() && probe->getProbeType() == scene::ReflectionProbe::ProbeType::PLANAR) {
+        if (probe->needRender()) {
             renderStage(camera, probe);
         }
     }
 }
 
 void ReflectionProbeFlow::renderStage(scene::Camera *camera, scene::ReflectionProbe *probe) {
     for (auto &stage : _stages) {
-        auto * framebuffer = probe->getRealtimePlanarTexture()->getWindow()->getFramebuffer();
-        auto *reflectionProbeStage = static_cast<ReflectionProbeStage *>(stage.get());
-        reflectionProbeStage->setUsage(framebuffer, probe);
-        reflectionProbeStage->render(camera);
-        probe->updatePlanarTexture(camera->getScene());
+        if (probe->getProbeType() == scene::ReflectionProbe::ProbeType::PLANAR) {
+            auto * framebuffer = probe->getRealtimePlanarTexture()->getWindow()->getFramebuffer();
+            auto *reflectionProbeStage = static_cast<ReflectionProbeStage *>(stage.get());
+            reflectionProbeStage->setUsage(framebuffer, probe);
+            reflectionProbeStage->render(camera);
+            probe->updatePlanarTexture(camera->getScene());
+        } else {
+            for (uint32_t faceIdx = 0; faceIdx < 6; faceIdx++) {
```

Comment:
It is better not use magic number. Or add some comment of the magic number is better.